nilearn/input_data/tests/test_nifti_masker.py

```diff
@@ -314,9 +315,18 @@ def test_compute_epi_mask():
                              get_data(mask4)[3:12, 3:12])
 
 
+@deprecated("Function 'test_compute_gray_matter_mask' has been renamed to "
+            "'test_compute_brain_mask' and "
+            "'test_compute_gray_matter_mask' will be removed in release 0.9.")
 def test_compute_gray_matter_mask():
     # Check masker for template masking strategy
 
+    return test_compute_brain_mask()
```

Comment:
I don't think that we need a deprecated test. This function can simply be removed.

Reply:
I have removed the deprecated test but not the `test_compute_brain_mask()`, if that's what you meant.